### PR TITLE
Bg grad Coding review #2

### PR DESCRIPTION
**Pull request by euqueme**

### CODE REVIEW

**Review by uche-inyama — COMMENTED** [review]
## Pull request under review by Inyama, Uchechukwu Henry

**Review by uche-inyama — CHANGES_REQUESTED** [review]
##  Hi Maru! great job :smile:
### Just few changes and you are good.

- [x] There is a validation error, with your Html, correct this.
- [x] Please fix the positioning of your images. I took a snap shot :point_down: 

Screenshot from 2019-09-03 20-55-16

### Implement these changes and submit another form for a code review :smiley: 

#### cheers Henry!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+          <a href="#"></a>
+        </li>
+        <li id="years-30" class="float-left">
+          <a href="#"></a>
+        </li>
+        <li id="supplier" class="float-left">
+          <a href="#"></a>
+        </li>
+      </ul>
+    </section>
```
Comment (samgaco) [inline index.html:70]: [SUGGESTION] Maybe the `<aside>` tag would fit better this part of the project.

> The <aside> element is used to identify content that is related to the primary content of the webpage, but does not constitute the primary content of the page. Author information, related links, related content, and advertisements are exampes of content that may be found in an aside element.


[Read more](https://html.com/tags/aside/#ixzz5yaXtnPH6)

**Review by samgaco — CHANGES_REQUESTED** [review]
#### Hi @euqueme :high_brightness: 

#### Everything is looking good and you went through the requested changes indicated by the previous TSE. Hence the project is approved :+1: Congratulations :100: 

---------

* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description


* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**